lib/utils/retry.go

```diff
@@ -155,6 +160,27 @@ func NewConstant(interval time.Duration) (*Linear, error) {
 	return NewLinear(LinearConfig{Step: interval, Max: interval})
 }
 
+// NewDefaultLinear creates a linear retry using a half jitter, 10s step, and maxing out
+// at 1 minute. These values were selected by reviewing commonly used parameters elsewhere
+// in the code base, which (at the time of writing) seem to converge on approximately this
+// configuration for "critical but potentially load-inducing" operations like cache watcher
+// registration and auth connector setup. It also includes an auto-reset value of 5m. Auto-reset
+// is less commonly used, and if used should probably be shorter, but 5m is a reasonable
+// safety net to reduce the impact of accidental misuse.
+func NewDefaultLinear() *Linear {
```

Comment:
Maybe add a TODO to actually use this throughout the codebase? Otherwise it's not a default, it's just the one used by the inventory control streams.